### DISCUSSION TITLE
My account flagged as spam help me (Emreodesia)

### DISCUSSION THREAD

**Discussion by MeyveliCorba** [community/community#145012] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Subject: Request for Reinstatement of My Account

Dear GitHub Support Team,

I hope this message finds you well. I am writing in response to the notice regarding my account being flagged. I would like to provide additional context and information that might assist in your review of my reinstatement request.

First and foremost, I would like to state that I have always adhered to GitHub’s community guidelines and terms of service to the best of my ability. If there has been any activity or content on my account that has caused concern, I assure you it was not intentional, and I am more than willing to cooperate fully in resolving the issue.

I am an active and engaged user of GitHub, utilizing the platform for open-source contributions and personal development under the username Emreodesia. My contributions have always been focused on positively engaging with the community, and I have never participated in or encouraged any activities that would be against the platform’s policies.

I respectfully request the opportunity to have my account reinstated, as it is an essential part of my workflow and professional growth. I understand the importance of maintaining a safe and collaborative environment on GitHub, and I am eager to comply with any necessary steps to rectify the situation.

Please let me know if there is any further information or clarification I can provide to assist in your review. I sincerely appreciate your time and consideration in resolving this matter.

Thank you for your attention to this issue. I look forward to hearing from you soon.

Best regards,
Emreodesia
https://github.com/Emreodesia

**Comment by Emreodesia** [comment]
My account flagged as spam  (Emreodesia)

I created a support request for this account but I haven't received an answer or response for months. I am a software enthusiast and a knowledgeable person, can you help me with this?

Emreodesia
https://github.com/Emreodesia

**Comment by xaviedoanhduy** [comment]
hi @MeyveliCorba, had a conversation about the same issue: https://github.com/orgs/community/discussions/122270

**Comment by uarix** [comment]
Hello, I'm sorry to hear that your account has been flagged. You can submit an appeal through the official channel, which typically takes about a week to process. Please note that community supporters cannot help expedite this process. In the meantime, you can check if your account is using an email service from an anonymous provider, if there are any unauthorized application permissions, or if there are any unknown repositories or other content added without your knowledge. If you find any of these issues, try removing them - this usually helps restore your account quickly.

**Reply by Emreodesia** [reply]
Thank you

**Comment by mecodeatlas** [comment]
Hi @MeyveliCorba!


This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
